hphp/runtime/vm/jit/vasm.h

```diff
@@ -107,6 +107,7 @@ bool checkBlockEnd(const Vunit& v, Vlabel b);
  */
 void allocateRegisters(Vunit&, const Abi&);
 void fuseBranches(Vunit&);
+void mapFlags(Vunit&);
```

Comment:
I'm not sure what "map" refers to.  I would call this something like `annotateSFUses()`, but any self-explanatory name is good.  (You can just name the file `vasm-flags.cpp`.)